edb/schema/objects.py

```diff
@@ -2031,6 +2031,13 @@ class InheritingObjectBase(Object):
         bool,
         default=False, compcoef=0.909)
 
+    def inheritable_fields(self) -> Iterable[Any]:
```

Comment:
I added two "abstract" methods, raising NotImplementedError, to explicit the expected protocol of this class, since later it's assumed that these two methods exist on the implementers. I am not sure you agree with this approach, but I think it makes sense.

Reply:
This is fine, thanks!